docs/specifications/tests/DNSQueryAndResponseDefaults.md

```diff
@@ -0,0 +1,162 @@
+
+A *DNSSEC Query* inherits the default setting from an *EDNS Query* except for the
+parameter specified below. If a *Parameter* is specified as "fixed" (with an
+"X" in that column) then the default value cannot be overidden.
+
+A test case specification can refer to a *DNSSEC Query* with one or several
+changes to the Parameters.
+
+|Parameter             |Default value |Fixed |Comment |
+|:---------------------|:-------------|:-----|:-------|
+|EDNS DO flag          | set          | X    |        |
+
+
+## Default handling of a *DNS Response*
+
+A *DNS Response* is a response to a *DNS Query*. Unless specified in the test
+case specification, the items in the response are handled as listed. If a
+*Response Item* is specified as "fixed" (with an "X" in that column) then the
+requirement, as specified under "Default handling", must be met for the response
+to be considered to be a DNS response.
+
+|Response Item |Default handling                          | Fixed | Comment              |
+|:-------------|:-----------------------------------------|:------|:---------------------|
+|OpCode        | Require value to be "response"           | X     |                      |
+|QR flag       | Require flag to be set                   | X     |                      |
+|AA flag       | -                                        |       | Defined in test case |
+|TC flag       | Re-query over TCP if set                 |       |                      |
+|RD flag       | ignore                                   |       |                      |
+|RA flag       | ignore                                   |       |                      |
+|AD flag       | ignore                                   |       |                      |
+|CD flag       | ignore                                   |       |                      |
+|RCODE         | -                                        |       | Defined in test case |
+|Query name    | ignore                                   |       |                      |
+|Query type    | ignore                                   |       |                      |
+|Query class   | Require value to be same as in the query |       | Normally "IN"        |
+|EDNS          | ignore                                   |       |                      |
+
+* Check against query name and query type is, by default, done against the values
+  in the query section in the query, not in the response.
+  
+* When fetching records from the answer section, these are the default criteria:
+  * Only records matching the query name and the query type are fetched. Any
+    RRSIG records meeting the next criterium are also fetched.
+  * RRSIG records with the same query name and covering the query type are
+    fetched.
+  * CNAME records are ignored unless the query type is CNAME.
+
+* When the test case specification states that CNAME chains are to be followed in
+  the the answer section, these are the default criteria for the chain to be
+  fetched:
```

Comment:
Since there are two defaults in play here I suggest we make that clear from the text.

```suggestion
* When the test case specification states that CNAME chains are to be followed,
  the default handling is to look for a valid CNAME chain in the answer section
  and fetch all CNAME records in that chain.
  These are the default criteria for a CNAME chain to be valid:
```